Apps/Sandcastle/gallery/CZML Reference Properties.html

```diff
@@ -10,10 +10,12 @@
     <script type="text/javascript" src="../Sandcastle-header.js"></script>
     <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
     <script type="text/javascript">
-    require.config({
-        baseUrl : '../../../Source',
-        waitSeconds : 60
-    });
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
```

Comment:
Should this be here? It's not in any of the other sandcastle examples.

Reply:
@ggetz that was generated when I saved the files directly from Sandcastle so I think it's intentional